docs/source/impl-guide/computed_identifiers.rst

```diff
@@ -236,6 +236,7 @@ Type prefixes used by VR are:
 
    SQ, Sequence
    VA, Allele
+   VS, VariationSet
```

Comment:
That's what I'd like in the future. However, TASC hasn't taken up this charge yet. So, for now, I recommend that we keep this is a VRS thing. Otherwise, they're going to have to create a mechanism for this review (and subsequent ones).

Reply:
I agree, but want GKS leads to weigh in too. Started up a conversation in Slack on #gks_leads.